app/src/main/java/com/dscvit/handly/util/Extensions.kt

```diff
@@ -0,0 +1,35 @@
+package com.dscvit.handly.util
+
+import android.content.Context
+import android.util.Patterns
+import android.view.View
+import android.widget.Toast
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+
+fun Context.shortToast(msg: String) {
+    Toast.makeText(this, msg, Toast.LENGTH_SHORT).show()
+}
+
+fun Context.longToast(msg: String) {
+    Toast.makeText(this, msg, Toast.LENGTH_LONG).show()
+}
+
+fun Fragment.shortToast(msg: String) {
+    Toast.makeText(this.requireContext(), msg, Toast.LENGTH_SHORT).show()
+}
+
+fun Fragment.longToast(msg: String) {
+    Toast.makeText(this.requireContext(), msg, Toast.LENGTH_LONG).show()
+}
+
+fun View.hide() {
+    this.isVisible = false
+}
+
+fun View.show() {
+    this.isVisible = true
+}
+
+fun CharSequence?.isValidEmail() =
```

Comment:
Since you are checking for nullness already, `isEmpty` should suffice instead of `isNullOrEmpty`.

Reply:
👍🏼